int64 bee_id
int64[] gene

